language/move-lang/tests/functional/validator_set/add_one_rotate_by_delegate.move

```diff
@@ -2,16 +2,15 @@
 // Test that alice can rotate bob's key and invoke reconfiguration.
 
 //! account: alice
-//! account: bob
+//! account: bob, 1000000, 0, validator
 //! account: carrol, 1000000, 0, validator
 
 //! sender: bob
 script {
 use 0x0::ValidatorConfig;
 fun main() {
-    ValidatorConfig::register_candidate_validator(x"beefbeef", x"10", x"20", x"30", x"40", x"50");
     // register alice as bob's delegate
-    ValidatorConfig::set_delegated_account({{alice}});
+    ValidatorConfig::rotate_operator({{alice}});
```

Comment:
I think just calling this "set_operator" is probably good... or even "set_config"

rotate feels like a phrase that implies that the client is sending only a request with no additional data. not a requirement, just a thought on this topic...

Reply:
That's a very good idea! I wanted to rename rotate_ to set_, but forgot...